### ISSUE TITLE
[Question] is it possible to hide the applet instead of collapsing?

### ISSUE THREAD

**Issue by buntupana** [psifidotos/applet-window-buttons#50]
Hi, I have enabled the option " Show when Active window is maximized" and the applet collapses when I don't have a maximized window. I wanted to know if it's possible to just hide the buttons keeping the applet's space so the other elements of the panel doesn't have to move.

Thanks, you are doing a great job.

**Comment by psifidotos** [comment]
It is not but I would like to see some screenshots for what you are trying to achieve

**Comment by trmdi** [comment]
I think he wants the fixed width for the applet in all cases.

**Comment by psifidotos** [comment]
Yeah I know, but I can't imagine a case that this would be really valued

**Comment by blueuser2** [comment]
I have a case where this would be very useful.
My top panel uses the window buttons, window title and window appmenu applets.
My appmenu replaces the window title when I hoover over it.
I have a launcher on the side (unity style) that pushes the appmenu entries out when the window controls are hidden.
If the window controls could leave a space behind then my appmenu would work just as well when my window is not maximised.
some screenshots:

![screenshot1](https://user-images.githubusercontent.com/55752317/65520723-69047b80-df2b-11e9-83a4-ad772f25581e.png)
![screenshot2](https://user-images.githubusercontent.com/55752317/65520765-791c5b00-df2b-11e9-9ddd-da988341bba9.png)
![screenshot3](https://user-images.githubusercontent.com/55752317/65520828-93eecf80-df2b-11e9-9fde-406babe3869a.png)

**Comment by buntupana** [comment]
> Yeah I know, but I can't imagine a case that this would be really valued

In my case I have a top bar with the applet next to the systray. So every time that I click in one element of the systray, all the elements of the bar got moved, which is a bit annoying. 

example: [https://media.giphy.com/media/U7n8L5NQkYkvHcuk29/giphy.gif](url)

Even when I minimized the windows all the elements I have in the bar are moved I would prefer to have them static in the same position.

**Comment by buntupana** [comment]
> Yeah I know, but I can't imagine a case that this would be really valued

In my case I have a top bar with the applet next to the systray. So every time that I click in one element of the systray, all the elements of the bar got moved, which is a bit annoying. [example](https://media.giphy.com/media/U7n8L5NQkYkvHcuk29/giphy.gif)

Even when I minimized the windows all the elements I have in the bar are moved in my case I prefer to have them static in the same position.
